vigor/templates/vigor/login.html

```diff
@@ -0,0 +1,11 @@
+<form method="post">
+  {% csrf_token %}
+  Username: <input type="text" name="username" value="" size="50" />
+  Password: <input type="password" name="password" value="" size="50" />
```

Comment:
I used that second URL as a reference, but I did not see an API for a login form. Only a signup form. Otherwise, the code is nearly identical.